examples/MODULE.bazel

```diff
@@ -19,7 +19,7 @@ protoc = use_extension("@toolchains_protoc//protoc:extensions.bzl", "protoc")
 protoc.toolchain(
     google_protobuf = "com_google_protobuf",
     # Demonstrate overriding the default version
-    version = "v28.0",
+    version = "LATEST",
```

Comment:
this no longer demonstrates how to override the default version. Can you change this to still pin the examples to a specific release?

Reply:
never mind I'll just revert after merging